app/controllers/admin/integrations_controller.rb

```diff
@@ -4,6 +4,18 @@ class Admin::IntegrationsController < Admin::AdminController
   def show
     @api_key = current_company.company_setting.api_key
     @intercom_access_token = current_company.company_setting.intercom_access_token
+    @company_setting = current_company.company_setting
+    @canvas_label = 'Share ideas or feedback'
+    @canvas_text = 'Share Feedback'
+    @canvas_label = @company_setting.intercom_canvas_settings['canvas_label'] if @company_setting.intercom_canvas_settings && !@company_setting.intercom_canvas_settings['canvas_label'].blank?
```

Comment:
`.present?` is what want here maybe? Do we only want to set it if it's present?

```suggestion
    @canvas_label = @company_setting.intercom_canvas_settings['canvas_label'] if @company_setting.intercom_canvas_settings && @company_setting.intercom_canvas_settings['canvas_label'].present?
```